### PR TITLE
Plans: Use sentence case for plan labels.

### LINKED ISSUE #40107: Plans: Plan descriptions should use sentence case for their bolded text.
[Reported](https://github.com/Automattic/wp-calypso/pull/40101#issuecomment-598461584) by @jancavan :

According to p9Jlb4-1ly-p2, we should use sentence case for plan description leading text (the title-like text in bold). These can be found at `/plans`.

Screen Shot 2020-03-12 at 5 14 59 PM

### PR DESCRIPTION
**Pull request by pento**
#### Changes proposed in this Pull Request

This PR changes the plan labels to use sentence case. See #40107 for rationale.

#### Testing instructions

* Visit `/plans/mysite.wordpress.com`
* Confirm that all of the plan "Best for..." labels use sentence case.

Fixes #40107.

### CODE REVIEW

File: client/lib/plans/plans-list.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const getPlanBloggerDetails = () => ( {
 	getStoreAudience: () => i18n.translate( 'Best for bloggers' ),
 	getDescription: () =>
 		i18n.translate(
-			'{{strong}}Best for Bloggers:{{/strong}} Brand your blog with a custom .blog domain name, and remove all WordPress.com advertising. Receive additional storage space and email support.',
+			'{{strong}}Best for bloggers:{{/strong}} Brand your blog with a custom .blog domain name, and remove all WordPress.com advertising. Receive additional storage space and email support.',
```
Comment (a8ci18n) [inline client/lib/plans/plans-list.js:34]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/lib/plans/plans-list.js [hunk 2]
```diff
@@ -79,7 +79,7 @@ const getPlanPersonalDetails = () => ( {
 	getStoreAudience: () => i18n.translate( 'Best for personal use' ),
 	getDescription: () =>
 		i18n.translate(
-			'{{strong}}Best for Personal Use:{{/strong}} Boost your' +
+			'{{strong}}Best for personal use:{{/strong}} Boost your' +
```
Comment (a8ci18n) [inline client/lib/plans/plans-list.js:82]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **37** times:
`translate( '{{strong}}Best for Personal Use:{{/strong}} Boost your website with a custom domain name, and remove all WordPress.com advertising. Get access to high-quality email and live chat support.' )` *ES Score: 10*

File: client/lib/plans/plans-list.js [hunk 3]
```diff
@@ -129,7 +129,7 @@ const getPlanEcommerceDetails = () => ( {
 	getStoreAudience: () => i18n.translate( 'Best for online stores' ),
 	getDescription: () => {
 		return i18n.translate(
-			'{{strong}}Best for Online Stores:{{/strong}} Sell products or services with this powerful, ' +
+			'{{strong}}Best for online Stores:{{/strong}} Sell products or services with this powerful, ' +
```
Comment (a8ci18n) [inline client/lib/plans/plans-list.js:132]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( '{{strong}}Best for Online Stores:{{/strong}} Sell products or services with this powerful, all-in-one online store experience. This plan includes premium integrations and is extendable, so it’ll grow with you as your business grows.' )` *ES Score: 12*

File: client/lib/plans/plans-list.js [hunk 4]
```diff
@@ -214,7 +214,7 @@ const getPlanPremiumDetails = () => ( {
 	getStoreAudience: () => i18n.translate( 'Best for freelancers' ),
 	getDescription: () =>
 		i18n.translate(
-			'{{strong}}Best for Freelancers:{{/strong}}' +
+			'{{strong}}Best for freelancers:{{/strong}}' +
```
Comment (a8ci18n) [inline client/lib/plans/plans-list.js:217]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( '{{strong}}Best for Freelancers:{{/strong}} Build a unique website with advanced design tools, CSS editing, lots of space for audio and video, Google Analytics support, and the ability to monetize your site with ads.' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/40107-sentence-case)

File: client/lib/plans/plans-list.js [hunk 5]
```diff
@@ -279,7 +279,7 @@ const getPlanBusinessDetails = () => ( {
 	getStoreAudience: () => i18n.translate( 'The plan for small businesses' ),
 	getDescription: () =>
 		i18n.translate(
-			'{{strong}}Best for Small Businesses:{{/strong}} Power your' +
+			'{{strong}}Best for small businesses:{{/strong}} Power your' +
```
Comment (a8ci18n) [inline client/lib/plans/plans-list.js:282]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( '{{strong}}Best for Small Businesses:{{/strong}} Power your business website with custom plugins and themes, unlimited premium and business theme templates, 200 GB storage, and the ability to remove WordPress.com branding.' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/40107-sentence-case)

File: client/lib/plans/plans-list.js [hunk 6]
```diff
@@ -680,7 +680,7 @@ export const PLANS_LIST = {
 		getPathSlug: () => 'premium',
 		getDescription: () =>
 			i18n.translate(
-				'{{strong}}Best for Small Businesses:{{/strong}}' +
+				'{{strong}}Best for small businesses:{{/strong}}' +
```
Comment (a8ci18n) [inline client/lib/plans/plans-list.js:683]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( '{{strong}}Best for Small Businesses:{{/strong}}Comprehensive, automated scanning for security vulnerabilities, fast video hosting, and marketing automation.' )` *ES Score: 12*

File: client/lib/plans/plans-list.js [hunk 7]
```diff
@@ -799,15 +799,15 @@ export const PLANS_LIST = {
...
+		getAudience: () => i18n.translate( 'Best for personal use' ),
 		getProductId: () => 2005,
 		getStoreSlug: () => constants.PLAN_JETPACK_PERSONAL,
 		availableFor: ( plan ) =>
 			includes( [ constants.PLAN_JETPACK_FREE, constants.PLAN_JETPACK_PERSONAL_MONTHLY ], plan ),
 		getPathSlug: () => 'jetpack-personal',
 		getDescription: () =>
 			i18n.translate(
-				'{{strong}}Best for Personal Use:{{/strong}}' +
+				'{{strong}}Best for personal use:{{/strong}}' +
```
Comment (a8ci18n) [inline client/lib/plans/plans-list.js:810]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( '{{strong}}Best for Personal Use:{{/strong}}Security essentials for your WordPress site, including automated backups and priority support.' )` *ES Score: 10*

File: client/lib/plans/plans-list.js (hunk 3 above)
Comment (aaronrobertshaw) [inline client/lib/plans/plans-list.js:132]: Should the "S" in "Stores" also be lowercase?

```suggestion
			'{{strong}}Best for online stores:{{/strong}} Sell products or services with this powerful, ' +
```

**Comment by pento** [comment]
Good catch, thanks @aaronrobertshaw! I double checked, this was the only one missed (searching for `Best for [a-z ]+[A-Z]`).

File: client/lib/plans/plans-list.js [hunk 8]
```diff
@@ -129,7 +129,7 @@ const getPlanEcommerceDetails = () => ( {
 	getStoreAudience: () => i18n.translate( 'Best for online stores' ),
 	getDescription: () => {
 		return i18n.translate(
-			'{{strong}}Best for Online Stores:{{/strong}} Sell products or services with this powerful, ' +
+			'{{strong}}Best for online stores:{{/strong}} Sell products or services with this powerful, ' +
```
Comment (a8ci18n) [inline client/lib/plans/plans-list.js:132]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( '{{strong}}Best for Online Stores:{{/strong}} Sell products or services with this powerful, all-in-one online store experience. This plan includes premium integrations and is extendable, so it’ll grow with you as your business grows.' )` *ES Score: 12*

**Comment by Copons** [comment]
This LGTM, but looking at it, I wonder if those labels actually need a colon. 🤔

**Comment by pento** [comment]
Added `String Freeze` to this PR, as this change causes `en` strings to show on the plans page.

@Automattic/i18n: Can these strings be fuzzy matched, since they're just case changes?

**Comment by pento** [comment]
> This LGTM, but looking at it I wonder if those labels actually need a colon. 🤔

@Copons: It would potentially work without the colon if we added some padding between the label and the paragraph. I'm inclined to leave it as is, though.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3598666

Thank you @pento for including a screenshot in the description! This is really helpful for our translators.

**Pull request merged**